src/runtime/io.cpp

```diff
@@ -690,11 +690,10 @@ extern "C" LEAN_EXPORT obj_res lean_io_remove_file(b_obj_arg fname, obj_arg) {
 
 extern "C" LEAN_EXPORT obj_res lean_io_app_path(obj_arg) {
 #if defined(LEAN_WINDOWS)
-    HMODULE hModule = GetModuleHandleW(NULL);
-    WCHAR path[MAX_PATH];
-    GetModuleFileNameW(hModule, path, MAX_PATH);
-    std::wstring pathwstr(path);
-    std::string pathstr(pathwstr.begin(), pathwstr.end());
+    HMODULE hModule = GetModuleHandle(NULL);
```

Comment:
```suggestion
    HMODULE hModule = GetModuleHandleA(NULL);
```
otherwise this does the wrong thing with `UNICODE` defined

Reply:
(and throughout)

Reply:
Don't define `UNICODE` then!